src/main/java/com/nu/art/http/HttpModule.java

```diff
@@ -217,7 +230,12 @@ long getTotalHoopTime() {
 		private HoopTiming hoop = new HoopTiming();
 
 		protected IHttpRequest createRequest() {
-			return new HttpRequestIn();
+			HttpRequestIn httpRequest = new HttpRequestIn();
+			if (defaultHeaders != null & defaultHeaders.size() > 0)
```

Comment:
This condition is also gone once created the instance at the declaration

Reply:
Removed the null check.